### PR TITLE
Remove mantid qt dir references

### PR DESCRIPTION
**Pull request by mantid-roman**
Removed from the build files references to the old MantidQt/Vates directories that I could find.

### CODE REVIEW

**Comment by martyngigg** [comment]
The `cppcheck` build has actually passed but the notification step timed out for some reason: http://builds.mantidproject.org/job/pull_requests-cppcheck/16025/console

File: qt/scientific_interfaces/test/EnggDiffractionPresenterTest.h [hunk 1]
```diff
@@ -124,12 +124,6 @@ class EnggDiffractionPresenterTest : public CxxTest::TestSuite {
     testing::NiceMock<MockEnggDiffractionView> mockView;
     MantidQt::CustomInterfaces::EnggDiffractionPresenter pres(&mockView);
 
-    // needs basic calibration settings from the user
```
Comment (martyngigg) [inline qt/scientific_interfaces/test/EnggDiffractionPresenterTest.h]: Why was a test removed?

Comment (mantid-roman) [inline qt/scientific_interfaces/test/EnggDiffractionPresenterTest.h]: The calls to the tested function were unnecessary in the implementations and I removed them. It looked as if they were there to satisfy the tests without doing anything useful.

Comment (martyngigg) [inline qt/scientific_interfaces/test/EnggDiffractionPresenterTest.h]: I see. Thanks.

**Comment by martyngigg** [comment]
Note that this also fixes a larger crop of cppcheck warnings that 1.80 seems to warn about now.

**Comment by martyngigg** [comment]
The Windows 7 build has also passed (http://builds.mantidproject.org/job/pull_requests-win7/16292/) but for some reason the status has not been updated.

**Comment by martyngigg** [comment]
@mantid-roman Is this okay to go now?

**Comment by mantid-roman** [comment]
It should be

**Review by martyngigg — APPROVED** [review]
Good to go.

The builds have actually passed but the notifications got stuck for some reason.

@mantidproject/gatekeepers LGTM

**Comment by martyngigg** [comment]
@mantidproject/gatekeepers Can we get this merged also to fix up  cppcheck properly after the directory moves?

**Pull request merged**